src/VisualStudio/Core/Test.Next/Remote/RemoteHostClientServiceFactoryTests.cs

```diff
@@ -108,7 +108,6 @@ private RemoteHostClientServiceFactory.RemoteHostClientService CreateRemoteHostC
         {
             workspace = workspace ?? new AdhocWorkspace(TestHostServices.CreateHostServices());
             workspace.Options = workspace.Options.WithChangedOption(RemoteHostOptions.RemoteHost, true)
-                                                 .WithChangedOption(RemoteHostOptions.RemoteHostTest, true)
                                                  .WithChangedOption(ServiceFeatureOnOffOptions.ClosedFileDiagnostic, LanguageNames.CSharp, true)
```

Comment:
It's your code :)   You wrote it so that it would check RemoteHostOption.RemoteHost :)

Regardless... i guess i'm not getting it.  Why did you need to add another option?  You say "I added remote host test option so that I don't touch remote host option", but i dont' understand why you want that.

Why not just have the single option and just have the test change that sole option?  What is the value in having two options?